templates/interface/RedHat.erb

```diff
@@ -83,7 +83,6 @@ IPV6_ADDR="<%= @ipv6addr %>"
 IPV6_DEFAULTGW="<%= @ipv6_defaultgw %>"
 <% end -%>
 <% end -%>
```

Comment:
TY, I'd also fix this end here, should be in line 76, after
 IPV6INIT="<%= @ipv6init %>"

I see the logic of having all the ipv6 options contained only if ipv6init is set, but I still prefer to treat this parameter as the others, indipendently.